integration-tests/apiGateway/index.ts

```diff
@@ -0,0 +1,109 @@
+// Copyright 2016-2019, Pulumi Corporation.
```

Comment:
2020! Although I guess you did open this in 2019 🤷‍♀

Reply:
🤷‍♂  ... I'll change it to 2020.